parsers/US_NY.py

```diff
@@ -8,13 +8,20 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas',              # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
+                                                # <5% is oil burn from 2016 State of NYISO Market Report
            'Natural Gas': 'gas',
            'Nuclear': 'nuclear',
-           'Other Fossil Fuels': 'unknown',
-           'Other Renewables': 'unknown',
+           'Other Fossil Fuels': 'coal',    # NYISO states "Power produced by either coal or oil" - assume coal, oil only power plants only as last resort
```

Comment:
Following up on https://github.com/tmrowco/electricitymap/pull/942#discussion_r159056769.
Let's add just a reference to document this. The 8260MW of installed oil capacity must come from somewhere (right now we assume it to be in "Other Fossil Fuels", so we need to be sure if we exclude it).

Reply:
Copy, just checked the largest oil only generator in NY - Oswego Harbor @1,804 MW - 2015 & 2016 it had a capacity factor of 0% as per Synapse Energy (Google Earth) that pulls information from US EIA.  As well as a few others Ravenswood & Roseton.  

I think what we have here is poor data on Generation Capacity that I originally pulled from Wikipedia https://en.wikipedia.org/wiki/List_of_power_stations_in_New_York  In reality these are "dual fuel" plants, so I've double counted in capacity.  I'll cleanup, which will drop Oil capacity to around 2,000 MW.

Would it be possible to split a real time data stream within the parser 50/50 into the 2 fuel types "coal" & "oil"?  Then if NYISO reports 1,500 MW under "other fossil fuels" the bars will show 750 MW of coal and 750 MW of oil?  Ultimately we are talking about 3,000 MW of combined coal & oil capacity - each with very low capacity factors over the year, against ~33,000 MW total installed.  The GWHr per year are a handful (less than 1%) compared to overall state generation of ~142,346 GWhr in 2015.

Reply:
It's possible but against our operating principles. Our philosophy is to try to do the least amount of assumptions as possible.
If there is an unknown is the data split, let's classify as unknown. This will also trigger visitors to ask questions and push on the data provider to provide better data.

Reply:
"Other Fossil Fuel" NYISO reporting = coal & oil
- Coal = 1,027 MW
- Oil - only = 2,877 MW
---> but US_NY parser cannot "split" single data streams into two fuel types "coal" & "oil"

"Natural Gas" NYISO reporting = natural gas
- NG - only = 3,898 MW
---> US_NY parser no issues attributing data properly as "gas"

"Dual Fuel" NYISO reporting = generators that can burn oil, gas or both together
- O&G - "Dual  Fuel" = 20,253 MW
---> US_NY parser, which way "gas" or "unknown"?

Data that says the parser registers "dual fuel" as "gas"....
- page 17, Table 2  note "fuel oil" is less than "coal"
- page A-8, 
"_Some generators in New York have dual-fuel capability, allowing them to burn either oil or natural gas. These generators usually burn the most economic fuel, although some may burn oil even when it is more expensive if natural gas is difficult to obtain on short notice or if there is uncertainty about its availability. In addition, New York City and Long Island reliability rules (which are discussed further in Section V.E) sometimes require that certain units burn oil in order to limit the exposure of the electrical grid to possible disruptions in the supply of natural gas. Since most large steam units can burn residual fuel oil (No.6) or natural gas, the effects of natural gas price spikes on power prices are partly mitigated by generators switching to fuel oil._"
- pg A-11, "_Average oil-fired generation fell substantially from 2015, primarily because lower natural gas prices in the first quarter of 2016 made oil production less economic._"
-pg A-180, Table A-11... note columns "Real Time Run Hours" greater than 99% of "Dual Fuel" units run on gas.  DF on Oil mostly zeros.  The few not "0" are in the double digits against year total of 8760 hours.  20 hrs = 0.228% and 100 hrs = 1.142%

Evidence supports listing "dual fuel" as "gas"

NG + Dual Fuel = 24,152 MW capacity under the "Gas" bar for US_NY parser

http://www.nyiso.com/public/webdocs/markets_operations/documents/Studies_and_Reports/Reports/Market_Monitoring_Unit_Reports/2016/NYISO_2016_SOM_Report_5-10-2017.pdf


Side point regarding Hydro & PS within the report.... in other words we are having generation kWh removed from the NYISO report equal to 2x the pumped storage operation... so 2 x "24"gCO2eq/kWh  If all PS is running 1,411 MW x 24g = 33,852 g/kW is "missing".   grrrrr

- pg A-9 "_**Pumped-storage resources in pumping mode are treated as negative generation.** The “Other” category includes methane, refuse, solar, and wood._"


sorry, have to jet but wanted to get this info out there.